go/logic/topology_recovery.go

```diff
@@ -93,35 +93,37 @@ type BlockedTopologyRecovery struct {
 type TopologyRecovery struct {
 	inst.PostponedFunctionsContainer
 
-	Id                        int64
-	UID                       string
-	AnalysisEntry             inst.ReplicationAnalysis
-	SuccessorKey              *inst.InstanceKey
-	SuccessorAlias            string
-	IsActive                  bool
-	IsSuccessful              bool
-	LostReplicas              inst.InstanceKeyMap
-	ParticipatingInstanceKeys inst.InstanceKeyMap
-	AllErrors                 []string
-	RecoveryStartTimestamp    string
-	RecoveryEndTimestamp      string
-	ProcessingNodeHostname    string
-	ProcessingNodeToken       string
-	Acknowledged              bool
-	AcknowledgedAt            string
-	AcknowledgedBy            string
-	AcknowledgedComment       string
-	LastDetectionId           int64
-	RelatedRecoveryId         int64
-	Type                      RecoveryType
-	RecoveryType              MasterRecoveryType
+	Id                         int64
+	UID                        string
+	AnalysisEntry              inst.ReplicationAnalysis
+	SuccessorKey               *inst.InstanceKey
+	SuccessorAlias             string
+	SuccessorBinlogCoordinates string
+	IsActive                   bool
+	IsSuccessful               bool
+	LostReplicas               inst.InstanceKeyMap
+	ParticipatingInstanceKeys  inst.InstanceKeyMap
+	AllErrors                  []string
+	RecoveryStartTimestamp     string
+	RecoveryEndTimestamp       string
+	ProcessingNodeHostname     string
+	ProcessingNodeToken        string
+	Acknowledged               bool
+	AcknowledgedAt             string
+	AcknowledgedBy             string
+	AcknowledgedComment        string
+	LastDetectionId            int64
+	RelatedRecoveryId          int64
+	Type                       RecoveryType
+	RecoveryType               MasterRecoveryType
 }
 
 func NewTopologyRecovery(replicationAnalysis inst.ReplicationAnalysis) *TopologyRecovery {
 	topologyRecovery := &TopologyRecovery{}
 	topologyRecovery.UID = util.PrettyUniqueToken()
 	topologyRecovery.AnalysisEntry = replicationAnalysis
 	topologyRecovery.SuccessorKey = nil
+	topologyRecovery.SuccessorBinlogCoordinates = ""
```

Comment:
initialize this as nil

Reply:
Thank you. I should have thought if this.